scenarios/ScenarioGoldNugget.py

```diff
@@ -0,0 +1,18 @@
+import random
+import scenarioManager
+    
+class ScenarioFoundNugget(scenarioManager.Scenario):
+    def __init__(self, gui, player, name, description, choice):
+        super().__init__(gui, player, name, description, choice)
+
+    def run(self):
+        print(f"Running scenario {self.name}...")
+        print(self.description)
+        
+        self.mod.distance = random.randint(75, 125)
+        print(f"You found a gold nugget in the stream. ${self.mod.distance} has been added to your cash")
```

Comment:
You should have a line that writes the result in the mod object like this
self.mod.result = f"You found a gold nugget in the stream. ${self.mod.distance} has been added to your cash"
Also make sure to include the f before the string